internal/jsonrpc2/chan_object_stream.go

```diff
@@ -0,0 +1,31 @@
+package jsonrpc2
+
+// This is mostly intented as a testing utility
```

Comment:
this is only used in tests so it doesn't matter if the implementation is inaccurate